File: javascript/challenge16.js
const festivalColours = color1 => {
  let saturation = ((color1 + 150) > 360 ) ? color1 - 210 : color1 + 150;
  let lightness = ((color1 + 210) > 360 ) ? color1 - 150 : color1 + 210;
  return [saturation, lightness].sort();
}

File: python/challenge16.py
def festivalColours(color1):
  saturation = (color1 - 210) if ((color1 + 150) > 360 ) else color1 + 150
  lightness = (color1 - 150) if ((color1 + 210) > 360 ) else color1 + 210;
  return [saturation, lightness].sort() 
